### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by plweil** [xaksis/vue-good-table#751]
### Issue Type (delete the irrelevant ones) 

Enhancement Request

### Specs
version 2.21.0

What browser?
Chrome 84.0.4147.135 

### Expected Behavior
Tables should meet WCAG accessibility guidelines. Search should have an accessible name; e.g., a label; column headers/sort "buttons" should be keyboard accessible; elements should have sufficient color contrast

### Actual Behavior
Sort is not accessible for keyboard/screen reader users; Search is missing a label; several elements fail WCAG color contrast requirements.

### Steps to Reproduce the Problem
Run vue-good-table through Axe, ARC Toolkit and similar accessibility testing tools. Or try using a screenreader, such as VoiceOver, to sort tables.

[EDIT]
I've submitted PR #752 to resolve (some of) these issues.

**Comment by xaksis** [comment]
PR has been merged. Thank you @plweil
